web-console/knikubevirt/windows-vms/windows-vms.md

```diff
@@ -0,0 +1,66 @@
+# Windows VMs
+
+The purpose of this PR is to research the details of a Windows VM vs any other option currently available via CNV. It will also try to cover how a Windows VM impacts each step of creating a VM flow, as well as after the VM has been created and also actions that  a user can take to improve the operability of the Windows VM.
```

Comment:
I think we could remove this paragraph entirely. It could be added to the description of the PR but really a title of End to end flow of creating a windows VM should cover this.